content/en/docs/installation/supported-releases.md

```diff
@@ -17,24 +17,27 @@ release every two months.
 
 ## Supported releases {#supported-releases}
 
-| Release | Release Date |        EOL        | [Supported Kubernetes versions][s] |
-| ------- | :----------: | :---------------: | :--------------------------------: |
-| [1.3][] | Apr 08, 2021 |   [~][]Aug 2021   | 1.16, 1.17, 1.18, 1.19, 1.20, 1.21 |
-| [1.2][] | Feb 10, 2021 | [~][]Jun 11, 2021 | 1.16, 1.17, 1.18, 1.19, 1.20, 1.21 |
+| Release | Release Date |     EOL      | [Supported Kubernetes versions][s] |
+|---------|:------------:|:------------:|:----------------------------------:|
+| [1.4][] | Jun 15, 2021 | Oct 13, 2021 | 1.16, 1.17, 1.18, 1.19, 1.20, 1.21 |
+| [1.3][] | Apr 08, 2021 | Aug 11, 2021 | 1.16, 1.17, 1.18, 1.19, 1.20, 1.21 |
 
 ## Upcoming releases
 
-| Release |   Release Date    |      EOL      | [Supported Kubernetes versions][s] |
-| ------- | :---------------: | :-----------: | :--------------------------------: |
-| [1.4][] | [~][]Jun 11, 2021 | [~][]Oct 2021 | 1.16, 1.17, 1.18, 1.19, 1.20, 1.21 |
+| Release | Release Date |     EOL      | [Supported Kubernetes versions][s] |
+|---------|:------------:|:------------:|:----------------------------------:|
+| [1.5][] | Aug 11, 2021 | Dec 15, 2021 | 1.16, 1.17, 1.18, 1.19, 1.20, 1.21 |
+| [1.6][] | Oct 13, 2021 | Feb 16, 2022 |           to be defined            |
+| [1.7][] | Dec 15, 2021 | Apr 13, 2022 |           to be defined            |
 
-> The `~` sign is used when the date is uncertain and might change; the
-> "EOL" abbreviation stands for End Of Life.
+> Note that dates in the future are are uncertain and might change. The "EOL"
```

Comment:
suggestion: remove duplicated "are"
```suggestion
> Note that dates in the future are uncertain and might change. The "EOL"
```

Reply:
this doesn't appear to be fixed in the latest version?

Reply:
Weird, I tried "Commit suggestion" twice and it does not seem to have create a commit... I ended up fixing the typo manually in 877e065.